avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/SchemaBuilder.java

```diff
@@ -28,20 +28,29 @@
 import joptsimple.OptionParser;
 import joptsimple.OptionSet;
 import joptsimple.OptionSpec;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 /**
  * generates java code from avsc files
  */
 public class SchemaBuilder {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(SchemaBuilder.class);
+
   private SchemaBuilder() { }
 
   public static void main(String[] args) throws Exception {
+    long start = System.currentTimeMillis();
+
+    long pluginLoadStart = System.currentTimeMillis();
     List<BuilderPlugin> plugins = loadPlugins(1);
+    long pluginLoadEnd = System.currentTimeMillis();
+    LOGGER.info("Loaded {} plugins in {} millis.", plugins.size(), pluginLoadEnd - pluginLoadStart);
 
+    long optionParseStart = System.currentTimeMillis();
```

Comment:
nit: do we need to log options parser? That is pretty standard

Reply:
Plugins can customize option parsing and some of them were doing a bad job perf wise before I cleaned it up. If we have a log, at least we will know if this regresses.

The issue is that since all this runs inside one gradle task, it is hard to look at build scans and figure out where time is being spent. These logs are our only help when debugging perf issues.